assets/src/components/nav/leftNav.tsx

```diff
@@ -32,6 +38,15 @@ const LeftNav = ({
   const [collapsed, setCollapsed] = useState<boolean>(defaultToCollapsed)
   const location = useLocation()
 
+  const bellIconClasses =
+    openView == OpenView.NotificationDrawer
+      ? [
+          "m-left-nav__icon",
+          "m-left-nav__icon--notifications-view",
+          "m-left-nav__icon--notifications-view--active",
```

Comment:
**praise:** I like calling it notifications _view_ for consistency, will have to move towards that terminology elsewhere in the source code.